File: dr Mustafa Saad/level-one/p10_Vanya_and_Fence.cpp
#include <iostream>
#include <vector>
using namespace std ;

/* Problem link : https://codeforces.com/contest/677/problem/A

    A. Vanya and Fence
    status link : https://codeforces.com/contest/677/submission/251663670
*/

int main(){

    // first line inputs 
    short friendNums = 1 ;
    float fenceHeight = 0 ;
    cin >> friendNums >> fenceHeight ;

    vector<float> pHeigths ;

    int result = friendNums ;

    // second line with checking 
    for (int i = 0; i < friendNums; i++) {
        float temp = 0 ;
        cin >> temp ;

        pHeigths.push_back(temp) ;

        if(pHeigths.at(i) > fenceHeight)  
            result += 1 ;
    }
     
     cout <<  result ;
}

File: dr Mustafa Saad/level-one/p10_Vanya_and_Fence.java

import java.util.Scanner ;

/* Problem link : https://codeforces.com/contest/677/problem/A

    A. Vanya and Fence
    status link : https://codeforces.com/contest/677/submission/251667359
*/
public class p10_Vanya_and_Fence {
    public static void main(String[] args ){
        Scanner input = new Scanner(System.in) ;

        short n = input.nextShort() ;
        float h = input.nextFloat() ;
        short result = 0 ;

        int[] persons = new int[n] ;

        result = n ;
        for(int i = 0 ; i < n ; ++i){
            persons[i] = input.nextInt() ;

            if(persons[i] > h)
                result += 1 ;

        }

        System.out.println(result);

    }
}